driver/modern_bpf/helpers/base/push_data.h

```diff
@@ -233,11 +234,19 @@ static __always_inline uint16_t push__charbuf(uint8_t *data, uint64_t *payload_p
 							(char *)charbuf_pointer);
 	}
 
-	if(written_bytes <= 0)
+	if(written_bytes < 0)
 	{
+		/* This is probably a page fault */
 		return 0;
 	}
 
+	/* Since `bpf_probe_read_user_str` return `0` in case of empty string we push a `\0` and we return 1. */
+	if(written_bytes==0)
```

Comment:
in case of an empty string, we send to userspace `""` instead of an empty param with len 0